src/main/java/baseball/exception/BaseballGameException.java

```diff
@@ -0,0 +1,46 @@
+package baseball.exception;
+
+import java.util.HashSet;
+
+public class BaseballGameException {
+    public void validPlayerNumIsInt(String playerNumStr) {
+        try{
+            Integer.parseInt(playerNumStr);
+        } catch (NumberFormatException e){
+            System.out.println("\n[Error] 세 자리 자연수를 입력하세요.");
+            throw new IllegalArgumentException();
+        }
+    }
+
+    public void validPlayerNumLenCount(String playerNumStr) {
+        if(playerNumStr.length() !=3){
+            System.out.println("\n[Error] 세 자리 수를 입력하세요.");
+            throw new IllegalArgumentException("");
```

Comment:
IllegalArgumentException 안에 메세지를 넣어 한번에 처리할 수 있을것 같아요

```suggestion
            throw new IllegalArgumentException("\n[Error] 세 자리 수를 입력하세요.");
```